lib/server.js

```diff
@@ -36,6 +37,7 @@ const STATS_INTERVAL = 5; // 5 seconds
 const STATS_EXPIRY = 30; // 30 seconds
 const statsClient = new StatsClient(localCacheClient, STATS_INTERVAL,
     STATS_EXPIRY);
+const enableRemoteManagement = true;
```

Comment:
will it be the default in master ?

Reply:
Yes the plan is to have opt-out. S3 Connector can use the env variable to disable though.